tests/components/timer/test_init.py

```diff
@@ -318,15 +318,10 @@ async def test_start_service(
     )
     await hass.async_block_till_done()
 
-    # Ensure an issue is raised for the use of this deprecated service
-    assert issue_registry.async_get_issue(
-        domain=DOMAIN, issue_id="deprecated_duration_in_start"
-    )
-
     state = hass.states.get("timer.test1")
     assert state
     assert state.state == STATUS_ACTIVE
-    assert state.attributes[ATTR_DURATION] == "0:00:15"
+    assert state.attributes[ATTR_DURATION] == "0:00:10"
```

Comment:
`ATTR_DURATION` refers to the configured `duration` which does not change

Reply:
That's confusing I think

Reply:
Isn't it enough with just the one attribute corresponding to the current duration?

Reply:
I thought it might be breaking but it's not so agree to convert it to the running duration which makes most sense.
It also means on idle timers this attribute will be `None` or optionally we set it to the configured value then.

Reply:
The state of an entity should always be the current duration, as that's the current value. We don't have to store default values in the state machine. 

At first thought, I would expect an idle timer to not have a duration (there is no timer running after all). However, I can also see having the default duration there. It means that if you call start without a duration, that's what will be applied.

Reply:
With the last commit it has now changed so if the timer is active (running or paused) the `duration` attribute refers to the running duration (which can be set by using the `duration` parameter for the `timer.start` service.
If the timer is idle (cancelled, finished) the `duration` attribute is the configured duration from the config entry.

`timer.change` service does not influence the duration. It just adds or subtracts to reach a new end-time.
Duration can only be changed by modifying the config entry (or YAML) or by `timer.start` service.
This hasn't changed. Just wanted to be very clear on this.

If you agree on this then I will make a doc to match with a note about the attribute.